resident-services/README.md

```diff
@@ -153,4 +162,18 @@
 
-* Registration Center List Location Hierarchy (Execution)  - This API will download a PDF of Registration Centers list. The Hierarchy level will be passed in this API along with its name from a file named as hirarchy_level and the variables used are "hierarchyLevel" and "name". The id and access token will be used here and can be used multiple times untill its valid and not expired. Hierarchy level are followed as 0:Country, 1:Region, 2:Province, 3:City, 4:Zone, 5:Postal Code.
\ No newline at end of file
+* Registration Center List Location Hierarchy (Execution)  - This API will download a PDF of Registration Centers list. The Hierarchy level will be passed in this API along with its name from a file named as hirarchy_level and the variables used are "hierarchyLevel" and "name". The id and access token will be used here and can be used multiple times untill its valid and not expired. Hierarchy level are followed as 0:Country, 1:Region, 2:Province, 3:City, 4:Zone, 5:Postal Code.
+
+* Track Service Request (Preparation) - This thread contains Auth Lock Unlock API which will generate Event ID in the response and will be used in the execution part. The Event ID will be stored in a file naming as eventid_service_request and can be used for multiple runs.
+
+* Track Service Request (Execution) -  This API is to get the details of status for a given Event ID and including its status. The Event ID generated in the preparation will passed from the file with the variable name as "eventId". The id and access token will be used here and can be used multiple times untill its valid and not expired.
+
+* Download Service Request (Execution) - This API is to convert the acknowledgement of any service request Event ID as a downloadable PDF. The Event ID stored in the file eventid_service_request can be used here to download the PDF. The id and access token will be used here and can be used multiple times untill its valid and not expired.
+
+* Channel Verification Status (Preparation) - This thread contains two APIs i.e Request OTP and Validate OTP which are added in a loop controller because we are running both the APIs for two type of cahnnels which are "EMAIL" and "PHONE". So will save the individual id and type of channel in a file naming channel_verification_list. The samples stored from the thread can be used multiple times untill OTP is valid and not expired.
+
+* Channel Verification Status (Execution) - This API is to check if OTP is verified for a channel type for an individual VID. The individual id and channel type will be passed in this API from a file named as channel_verification_list and the variable used are "individualIdVerification" and "channelVerification". The samples from the file can be used for multiple runs untill the OTP expiry time is valid.
```

Comment:
"The samples from the file can be used for multiple runs untill the OTP expiry time is valid." - How is the OTP expiry time  relevant ?

Reply:
In the preparation we are using request OTP API which is generating OTP so for that reason have mentioned that till the OTP is not expired it can be used for multiple runs.

Reply:
But we are mentioning this in the execution, it is confusing.

Reply:
As discussed, removed.